os_net_config/impl_nmstate.py

```diff
@@ -152,7 +152,7 @@ def parse_bonding_options(bond_options_str):
 
 
 def set_linux_bonding_options(bond_options, primary_iface=None):
-    linux_bond_options = ["updelay", "miimon", "lacp_rate"]
+    linux_bond_options = ["updelay", "miimon", "lacp_rate", "fail_over_mac"]
```

Comment:
Since we allow only this subset of bond options, can a warning be added if user providers other bond options like: downdelay, lacp_rate, balance_slb, etc ?

Reply:
More importantly can we add support for additional Linux bond options? downdelay and lacp_rate should be supported. In fact we should support the whole array of options that are supported in the nmstate schema.

Reply:
i will add to the linux_bond_options list all supported options. (btw options which are not mode that not include in the list will be ignored ... so 'downdelay' is currently not supported)